File: app/src/main/java/com/kra/paypoint/data/remote/model/auth/BhfUserSaveReq.kt
package com.kra.paypoint.data.remote.model.auth

import com.google.gson.annotations.SerializedName

data class BhfUserSaveReq(
    @SerializedName("tin") val tin: String,
    @SerializedName("bhfId") val bhfId: String,
    @SerializedName("userId") val userId: String,
    @SerializedName("userNm") val userNm: String,
    @SerializedName("pwd") val pwd: String?,
    @SerializedName("adrs") val adrs: String?,
    @SerializedName("cntc") val cntc: String?,
    @SerializedName("authCd") val authCd: String,
    @SerializedName("remark") val remark: String?,
    @SerializedName("useYn") val useYn: String,
    @SerializedName("regrId") val regrId: String?,
    @SerializedName("regrNm") val regrNm: String?,
    @SerializedName("modrId") val modrId: String?,
    @SerializedName("modrNm") val modrNm: String?
)

File: scratch/EBM2x_legacy/RraSdc/model/BhfUserSaveReq.cs
using System;
using System.Collections.Generic;

namespace EBM2x.RraSdc.model
{
    public class BhfUserSaveReq {
	    public string tin { get; set; }  // TIN
	    public string bhfId { get; set; }  // Branch ID
	    public string userId { get; set; }  // User ID
	    public string userNm { get; set; }  // User Name
	    public string pwd { get; set; }  // Password
	    public string adrs { get; set; }  // Address
	    public string cntc { get; set; }  // Contact
	    public string authCd { get; set; }  // Authority Code
	    public string remark { get; set; }  // Remark
	    public string useYn { get; set; }  // Used / UnUsed
        public string regrId { get; set; }  //   등록자 아이디
        public string regrNm { get; set; }  //    등록장 명
        public string modrId { get; set; }  //   수정자 아이디
        public string modrNm { get; set; }  //    수정자 명
    }
}
